Transition from action inputs to payment calculation

jaucourt opened the pull request:
https://eaflood.atlassian.net/browse/FCEP-53

Displays payment for given action inputs, or informs land manager that
no payment is applicable

jaucourt committed:
Transition from action inputs to payment calculation

https://eaflood.atlassian.net/browse/FCEP-53

Displays payment for given action inputs, or informs land manager that
no payment is applicable

jaucourt committed:
Updated contributor list

jaucourt committed:
Added payments service

jaucourt committed:
Refactored payments service to match expected request payload

jaucourt committed:
Added payments model

jaucourt committed:
Merge branch 'master' into feature/FCEP-43-transition-from-action-inputs-to-payment-calculation

jaucourt committed:
Front end display of calculation result

jaucourt committed:
Changed test name from copy-and-paste special

jaucourt committed:
Added paymentCalculationUrl to helm values

Matthew-Collins-Defra committed:
endPoint fix

jaucourt committed:
Fixes:
- pass request to getParcelRef / getActionId / getActionInput
- closing quote on class in template

jaucourt committed:
Merge branch 'feature/FCEP-43-transition-from-action-inputs-to-payment-calculation' of github.com:DEFRA/ffc-ce-web into feature/FCEP-43-transition-from-action-inputs-to-payment-calculation

jaucourt committed:
Switch to using post rather than get for calculating payments

neilbmclaughlin commented on test/services/payments-service.test.js:
```diff
@@ -0,0 +1,100 @@
+      expect(result).toEqual(payload)
+    })
+
+    const getSampleRequestPayload = (payload = {}) => ({
```
There are no calls to `getSampleRequestPayload` which pass a parameter.

neilbmclaughlin reviewed (commented):
LGTM

jaucourt committed:
Fixes:
- Update endpoint url
- Fix setActionInput session function
- Review fix on getSampleRequestPayload

neilbmclaughlin reviewed (approved):
LGTM
